### DISCUSSION TITLE
so-elastalert error post 2.3.10 upgrade

### DISCUSSION THREAD

**Discussion by endpointsec** [Security-Onion-Solutions/securityonion#2191] [Unsupported Versions] [answered]
Good morning, successfully applied 2.3.10 update today but now so-elastalert stays down (error). All other services are ok.

No log data inside /opt/so/log/elastalert/

Any ideas how to investigate what is causing elastalert to not spin up?

*edit* Found one of the customized playbook YAML files had some messed up formatting for the email alert. Addressed and restarted service with no further problems. May be closed.

**Comment by dougburks** [comment, marked as answer]
Marking as answered.
